app/models/sample.rb

```diff
@@ -30,7 +38,14 @@ def pipeline_command
     command
   end
 
-  def kickoff_pipeline(dry_run = true) # should be triggered when the upload is complete
+  def check_status
+    if [STATUS_UPLOADED, STATUS_RERUN].include?(status)
+      status = STATUS_CHECKED
```

Comment:
rubocop is warning about this, because you might need a `self.` here

Reply:
thanks. i will fix this. sometime rubcop wants me to do self. sometimes doesn't